motr/m0crate/crate.c

```diff
@@ -1254,6 +1257,33 @@ static void cr_btree_delete(struct m0_key_val *kv)
 			  bo_u.u_btree.t_rc);
 }
 
+static void cr_btree_warmup(struct cr_workload_btree *cwb)
+{
+	char 	v[cwb->cwb_max_val_size];
+	struct 	cr_btree_key cbk;
+
+	int 	i	= 0;
+	int 	count	= cwb->cwb_warmup_insert_count;
+	int 	ksize	= cwb->cwb_key_size != -1 ? cwb->cwb_key_size :
+			  getrnd(1, cwb->cwb_max_key_size);
```

Comment:
change it to  `getrnd(sizeof cbk.bkey, cwb->cwb_max_key_size)`

Reply:
Updated in the latest commit